src/EnergyPlus/DataSurfaces.hh

```diff
@@ -1509,6 +1530,11 @@ struct SurfacesData : BaseGlobalStruct
     Array1D<bool> SurfIsPool;                       // true if this is a pool
     Array1D<int> SurfICSPtr;                        // Index to ICS collector
     Array1D<bool> SurfIsRadSurfOrVentSlabOrPool;    // surface cannot be part of both a radiant surface & ventilated slab group
+    Array1D<bool> IsSurfPropertyGndSurfacesDefined; // true if ground surfaces properties are listed for an external surface
+    Array1D<int> GroundSurfsPropertyNum;            // index to a ground surfaces list (defined in SurfaceProperties::GroundSurfaces)
+    Array1D<bool> UseSurfPropertyGndSurfTemp;       // true if at least one ground surface temperature schedules is specified
+    Array1D<bool> UseSurfPropertyGndSurfRefl;       // true if at least one ground surfaces reflectance schedule is specified
+    Array1D<Real64> GndReflSolarRad;                // ground surface reflected solar radiation on exterior surfaces
```

Comment:
Surface variables for use exterior surface and multiple ground surfaces property model.

Reply:
So all 5 of these new `Array1D` have to be allocated and managed for each surface in the model?  Why not just add scalar variables to the surface data struct?  Any reason this is better?